doc/man3/ASN1_TIME_set.pod

```diff
@@ -199,9 +202,9 @@ ASN1_TIME_normalize() returns 1 on success, and 0 on error.
 ASN1_TIME_check(), ASN1_UTCTIME_check and ASN1_GENERALIZEDTIME_check() return 1
 if the structure is syntactically correct and 0 otherwise.
 
-ASN1_TIME_print(), ASN1_UTCTIME_print() and ASN1_GENERALIZEDTIME_print() return
-1 if the time is successfully printed out and 0 if an error occurred (I/O error
-or invalid time format).
+ASN1_TIME_print(), ASN1_UTCTIME_print() and ASN1_GENERALIZEDTIME_print()
+return 1 if the time is successfully printed out and
+0 if an I/O error occurred an error occurred (I/O error or invalid time format).
```

Comment:
kind of copy/paste mistake on this line.
Sorry for the delayed reaction.

Reply:
Thanks @FdaSilvaYY for pointing this out.
I've just simply added a small commit to #13735 for fixing the glitch.